go/vt/vtctl/workflow/server.go

```diff
@@ -2722,6 +2776,56 @@ func (s *Server) DropTargets(ctx context.Context, ts *trafficSwitcher, keepData,
 	return sw.logs(), nil
 }
 
+// DeleteTenantData attempts to delete all of the tenant's data that was migrated
+// in the workflow that we are canceling or deleting. This work can take some
+// time so if the context ends then the user will need to retry.
+func (s *Server) DeleteTenantData(ctx context.Context, ts *trafficSwitcher, batchSize int64) error {
```

Comment:
Why does this need to be exported? It doesn't seem to be called outside this package. DropTargets doesn't need to be exported either.

Reply:
Good catch! Neither of them need to be and I'll change that.